pgmoon/init.lua

```diff
@@ -964,7 +964,11 @@ do
             end
             return self.sock:sslhandshake(false, nil, self.config.ssl_verify)
           else
-            return self.sock:tlshandshake({ verify = self.config.ssl_verify, client_cert = luasec_opts.certificate, client_priv_key = luasec_opts.key })
+            if self.sock.tlshandshake then
```

Comment:
Without this change, the tests fail with https://github.com/Kong/pgmoon/runs/7117841709?check_suite_focus=true

Reply:
@Suika-Kong

Reply:
I found that `sslhandshake()` is not from the socket module but is assigned by pgmoon. And `tlshandshake()` is not assigned anywhere, nor is it from the socket module. I need some time to recall why I wrote that in the first place.

Reply:
And you need to also change the `init.moon`.

Reply:
It seems we have removed the `tlshandshake()` function for `nginx` socket.
And what surprises me more is that `sslhandshake()` receives `opts` table for `luasocket`(what we use for init phase) but we call it with 3 arguments.

Reply:
https://github.com/Kong/pgmoon/pull/22/commits/67dc4318721e84075b431273ceaea940016bb68e